doc/contributing/PROCESS.md

```diff
@@ -174,6 +174,92 @@ To make a new OpenAssetIO release, follow this procedure.
   uploaded correctly.
 - You're done! Take a break and relax!
 
+## Breaking Integrations
+
+OpenAssetIO is pursuing an integration heavy validation strategy, it
+seeks to have a set of robust proof of concept projects, and will use
+these as both design validators and testing coverage providers.
+
+In order to tie this coverage into mainline OpenAssetIO development,
+OpenAssetIO continuous integration checks out several downstream
+projects each time a commit is pushed. These projects are built against
+the bleeding edge version of OpenAssetIO, and their test suites are run.
+This provides valuable end to end testing of each changeset, and is
+invaluable. (See
+[integrations.yml](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/.github/workflows/integrations.yml))
```

Comment:
[consider] Using full URLs is more portable on the web, but if we assume we're not going to host this .md file elsewhere, then it's better to use relative paths.  

E.g. my IDE can follow relative paths in the docs when I view them locally in a markdown preview.

Reply:
Hmm, guess this was workflow bias, i tend to view code in the browser quite a lot even when i've got it available locally, will change over to relative paths.

Reply:
:pray:  selfishly, this serves me better as I often use the link to "open the right file to edit" (for this, and more generally in process documentation).